backend/open_webui/main.py

```
@@ -1107,44 +2510,57 @@ async def oauth_login(provider: str, request: Request):
-                "src": "/static/logo.png",
-                "type": "image/png",
-                "sizes": "500x500",
-                "purpose": "any",
-            },
-            {
-                "src": "/static/logo.png",
-                "type": "image/png",
-                "sizes": "500x500",
-                "purpose": "maskable",
+    if app.state.EXTERNAL_PWA_MANIFEST_URL:
+        return requests.get(app.state.EXTERNAL_PWA_MANIFEST_URL).json()
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)